bpfd/src/main.rs

```diff
@@ -53,23 +51,31 @@ fn main() -> anyhow::Result<()> {
             );
             setrlimit(Resource::RLIMIT_MEMLOCK, RLIM_INFINITY, RLIM_INFINITY).unwrap();
 
-            create_dir_all(BPFFS).context("unable to create mountpoint")?;
-            create_dir_all("/var/run/bpfd/programs")?;
-            create_dir_all("/var/run/bpfd/dispatchers")?;
+            create_dir_all(RTDIR)?;
+            create_dir_all(RTDIR_FS).context("unable to create mountpoint")?;
```

Comment:
Any reason for only adding context to one of these create calls?

Reply:
That's the way the existing code was, I just added more directories and moved them behind const strings.

Reply:
Wanna just go and add context to all of them it could be a nicer error message if ever encountered :)